tests/backend_test_base.py

```diff
@@ -83,47 +93,74 @@ def run_test_case(self, feed_dict, input_names_with_port, output_names_with_port
         # optional - pass distinct feed_dict to onnx runtime
         if onnx_feed_dict is None:
             onnx_feed_dict = feed_dict
-
+        input_names_with_port = list(feed_dict)
+        tf_reset_default_graph()
         graph_def = None
-        if convert_var_to_const:
-            with tf.Session() as sess:
-                tf.tables_initializer().run()
-                variables_lib.global_variables_initializer().run()
-                output_name_without_port = [n.split(':')[0] for n in output_names_with_port]
-                graph_def = tf.graph_util.convert_variables_to_constants(sess, sess.graph_def,
-                                                                         output_name_without_port)
 
-            tf.reset_default_graph()
+        np.random.seed(1)  # Make it reproducible.
```

Comment:
Is this only for certain test cases, or needed for tf 2.0?

Reply:
We already initialize the  tf.set_random_seed before calling func() (both for tf-1.x and tf-2.x) and I thought its good to do the same to numpy in case one wants to use np.random.* in func.